### ISSUE TITLE
Search Results Custom Loop is not appearing

### ISSUE THREAD

**Issue by JamesCooperRTW** [dudaster/ele-custom-skin#203]
Hi!

There seems to be a glitch when having multiple posts widget with custom skin. If you check this page and screenshot
https://switchautomation.rtwtestserver.info/?post_type=tools&s=switch
http://prntscr.com/sbpg16

But if you check the last post widget, it's using elementor default but the post appears. 

Let me know what the issue is. It only happens on search, on other pages, there are multiple post widget but the post contents has no issue.

Thanks,
James

**Comment by dudaster** [comment]
You are right. In archive the content disappears. Can't you use post summary or is it something more complicated?
